hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/BrokersClusterService.java

```diff
@@ -123,6 +139,19 @@ public Optional<ConsumerGroup> describeConsumerGroup(Topic topic, String subscri
         return consumerGroupsDescriber.describeConsumerGroup(topic, subscriptionName);
     }
 
+    public void moveOffsetsToTheEnd(Topic topic, SubscriptionName subscription) {
+        validateIfOffsetsCanBeMoved(topic, subscription);
+
+        KafkaConsumer<byte[], byte[]> consumer = kafkaConsumerManager.createConsumer(subscription);
+        String kafkaTopicName = kafkaNamesMapper.toKafkaTopics(topic).getPrimary().name().asString();
+        Set<TopicPartition> topicPartitions = getTopicPartitions(consumer, kafkaTopicName);
+        consumer.assign(topicPartitions);
+
+        Map<TopicPartition, Long> endOffsets = consumer.endOffsets(topicPartitions);
+        Map<TopicPartition, OffsetAndMetadata> endOffsetsMetadata = buildOffsetsMetadata(endOffsets);
+        consumer.commitSync(endOffsetsMetadata);
```

Comment:
I'm not sure, but shouldn't we call close on kafkaConsumer here?

Reply:
true ;)